rules/windows/process_creation/proc_creation_win_termserv_proc_spawn.yml

```
@@ -6,7 +6,7 @@ references:
     - https://securingtomorrow.mcafee.com/other-blogs/mcafee-labs/rdp-stands-for-really-do-patch-understanding-the-wormable-rdp-vulnerability-cve-2019-0708/
 author: Florian Roth (Nextron Systems)
 date: 2019/05/22
-modified: 2023/01/25
+modified: 2023/02/03
```

```suggestion
modified: 2023/01/25
```
we shouldn't change the modified date for cosmetic changes